cartographer/mapping/map_builder_interface.h

```diff
@@ -92,7 +92,7 @@ class MapBuilderInterface {
   // Loads the SLAM state froma a pbstream file. Returns the remapping of new
   // trajectory_ids.
   virtual std::map<int /* trajectory id in proto */, int /* trajectory id */>
-  LoadStateFromFile(const std::string& filename) = 0;
+  LoadStateFromFile(const std::string& filename, bool load_frozen_state) = 0;
```

Comment:
Note: changes the public API, but becomes consistent with LoadState

Reply:
That's why I changed it